### PR TITLE
New localized version of kitchensink-ear quickstart

### PR DESCRIPTION
**Pull request by sgilda**
What more is there to say? ;-)

### CODE REVIEW

**Comment by sgilda** [comment]
Hold off on this one. Something is not quite right here.

**Comment by sgilda** [comment]
I think this one is OK now.

File: kitchensink-ml-ear/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+kitchensink-ml-ear: Localized Example Using Multiple Java EE 6 Technologies Deployed as an EAR
+==============================================================================================
+Author: Pete Muir
```
Comment (pmuir) [inline kitchensink-ml-ear/README.md:3]: Wrong author ;-)

**Comment by pmuir** [comment]
Merged

**Pull request closed without merging**